src/appkit.cs

```diff
@@ -6935,6 +6953,13 @@ partial interface NSFont : NSSecureCoding, NSCopying {
 		[Internal]
 		IntPtr _SystemFontOfSize (nfloat fontSize, nfloat weight);
 
+		[Advice ("Use constants from 'NSFontWeight' and 'NSFontWidth'.")]
```

Comment:
I believe [Advice] is useless on an [Internal] method.

```suggestion
		[Advice ("Use constants from 'NSFontWeight' and 'NSFontWidth'.")]
```